### DISCUSSION TITLE
Does the filtering function support the use of wildcards?

### DISCUSSION THREAD

**Discussion by breakTBB** [aropan/clist#315] [Q&A] [answered]
Can wildcards be used to filter contest titles in the "contest" field in this [link](https://clist.by/problems/?search=&resource=93&rating_from=1200&rating_to=1400&status=&chart=&sort_order=desc&sort=rating), such as "AtCoder Beginner Contest *", to filter out all ABC contests? Currently, only exact matches are supported.
![image](https://github.com/aropan/clist/assets/16484507/e1c57e0e-947b-4957-8547-f10de4eae2ba)

**Comment by aropan** [comment, marked as answer]
Use this help:
<img width="593" alt="image" src="https://github.com/aropan/clist/assets/1968460/36a0ffcd-ec3e-4c3a-a493-faf87ef90349">

And check [here](https://clist.by/problems/?search=contest:AtCoder+Beginner+Contest&resource=93&rating_from=1200&rating_to=1400&sort_order=desc&sort=rating).
